tests/nlp/nlp_conftest.py

```diff
@@ -83,7 +84,20 @@ def movie_reviews_data_negative():
 @pytest.fixture(scope='session')
 def text_token_classification_dataset_mock():
     """Mock for a token classification dataset"""
-    return TextData(['Mary had a little lamb', 'Mary lives in London and Paris',
+    return TextData(raw_text=['Mary had a little lamb', 'Mary lives in London and Paris',
                      'How much wood can a wood chuck chuck?'],
-                    [[('B-PER', 0, 4)], [('B-PER', 0, 4), ('B-GEO', 14, 20), ('B-GEO', 25, 30)], []],
+                    label=[['B-PER', 'O', 'O', 'O', 'O'], ['B-PER', 'O', 'O', 'B-GEO', 'O', 'B-GEO'],
+                     ['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O']],
                     task_type='token_classification')
+
+@pytest.fixture(scope='session')
+def wikiann():
+    """Wikiann dataset for token classification"""
+    dataset = load_dataset('wikiann', name='en', split='train')
```

Comment:
Can we load less data?

Reply:
It loads 10,000 short sentences. Not a lot IMO